google-sheets/assets/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <style>
+      body {
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      div[role="main"] {
+        flex: 1;
+      }
+
+      footer {
+        text-align: center;
+      }
+
+      footer p {
+        margin-bottom: 0;
+      }
+
+      #twilio-logo {
+        width: 50px;
+        height: 50px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to your new Twilio App</h1>
+    </header>
+    <div role="main">
+      <section>
+        <h3>Get started with your app</h3>
+        <p>This app automatically logs texts to your phone number into the specified Google Sheets document. Follow the steps below to test your app:</p>
+        <ol>
+          <li>
+              <a href="https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating">
+                  Create a Google Cloud service account
+              </a>
+              to allow this app to add to your spreadsheet.
+          </li>
+          <li>
+            <a href="https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys">
+              Generate keys for your Google Cloud service account in JSON format
+            </a>
+            and save the resulting file as <code>auth.json</code> under this app's <code>assets</code> directory.
+          </li>
+          <li>Share your Google Sheets document with the email address of the service account.</li>
+          <li>Text your Twilio phone number with a message.</li>
+          <li>Your spreadsheet should now contain the SID, phone number, and content of the text you sent.</li>
+        </ol>
+      </section>
+      <section>
+        <h3>Troubleshooting</h3>
+        <ul>
+          <li>Check the <a href="https://www.twilio.com/console/phone-numbers/incoming">phone number configuration</a> and make sure the Twilio phone number you want your app has an SMS webhook configured to point at
+            <p>
+              <pre><code><span class="function-root"></span>/log-sms</code></pre>
+            </p>
+          </li>
+          <li>Ensure that you have created a Google service account, saved its private key as <code>auth.json</code> in your function's assets, and shared the spreadsheet you want to log into with that service account's email address as an Editor.</li>
+        </ul>
+      </section>
+    </div>
```

Comment:
this will probably need the `<!-- APP_INFO -->` comment to render the configured phone number